news/_posts/2021-02-22-Singular.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Singular.jl 0.4.5 released
+author: Dan Schultz   
+---
+Here is an overview of the changes in this release of [Singular.jl](https://github.com/oscar-system/Singular.jl):
+
+- Singular.jl should no longer crash on the new rings that Singular creates, or at least it should crash in far fewer cases.
```

Comment:
I think this might be a bit misleading, as it doesn't say what kind of rings this talks about: I think only rings created by library functions are affected by this?